src/test/App.test.js

```diff
@@ -3,21 +3,46 @@ import React, {Component} from 'react';
 import {shallow, mount, render} from 'enzyme'
 
 
-describe('App js shouls say hello world', ()=>{
+describe('App js shouls say Keep Ahead', ()=>{
+    let renderWrapper
+    let wrapper
+    let mountWrapper
 
-    const renderWrapper = render(< App/>)
-
-    it('should say hello', ()=>{
-        expect(renderWrapper.find('div h1').text()).toBe('Hello, React!')
+    beforeEach(() => {
+      renderWrapper = render(< App/>)
+      wrapper = shallow(< App/>)
+      mountWrapper = mount(< App/>)
     })
 
-    const wrapper = shallow(< App/>)
+    it('should display the app name', ()=>{
+        expect(renderWrapper.find('div h1').text()).toBe('Keep Ahead')
+    })
 
     it('should say hello', ()=>{
         expect(
             wrapper.containsMatchingElement(
-                <h1>Hello, React!</h1>
+                <h1>Keep Ahead</h1>
             )
         ).toBeTruthy()
     })
```

Comment:
We had issues with toBeTruthy comparing elements, as a variable will be truthy if it has value, did this issue come up for you and if so how did you combat it?